lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -956,7 +973,148 @@ section.instructor-dashboard-content-2 {
       }
     }
   }
+
+  /*
```

Comment:
FWIW, I'm going to review this code to be thorough. Please take my comments in mind as you refactor.